sections/cart-notification-product.liquid

```diff
@@ -20,6 +20,23 @@
                 <dd>{{ option.value }}</dd>
               </div>
             {%- endfor -%}
+            {%- for property in item.properties -%}
```

Comment:
Since all this markup is inside the `{%- unless item.product.has_only_default_variant -%}`, does that mean if the product has line item properties then `item.product.has_only_default_variant` will output `false`? Asking because otherwise the line item properties will not be displayed for products with only 1 variant?